examples/green_screen/MultiDeviceCapturer.h

```diff
@@ -0,0 +1,177 @@
+#pragma once
+
+#include <chrono>
+#include <vector>
+#include <k4a/k4a.hpp>
+
+// This is the maximum difference between when we expected an image's timestamp to be and when it actually occurred.
+constexpr std::chrono::microseconds MAX_ALLOWABLE_TIME_OFFSET_ERROR_FOR_IMAGE_TIMESTAMP(50);
+
+struct MultiDeviceCapturer
```

Comment:
Why struct and not class?

Reply:
I used to expose the devices vector at the bottom. I've just refactored to make it a class.